image-swipe.ios.ts

```diff
@@ -300,13 +302,15 @@ export class ImageSwipe extends ImageSwipeBase {
 
     private _calcScrollViewContentSize() {
         const scrollView: UIScrollView = this.nativeViewProtected;
-        const width = utils.layout.toDeviceIndependentPixels(this.getMeasuredWidth());
-        const height = utils.layout.toDeviceIndependentPixels(this.getMeasuredHeight());
-        const safeAreaInsets = this.getSafeAreaInsets();
-        const insetAllowance = utils.layout.toDeviceIndependentPixels(safeAreaInsets.left + safeAreaInsets.right);
-        const calculatedWidth = width + insetAllowance;
-
-        scrollView.contentSize = CGSizeMake(this.items.length * calculatedWidth, height);
+        if (scrollView) {
```

Comment:
Let's just reverse the logic and short circuit execution with a `return` here so we don't have to surround the whole function body.

Reply:
I agree